pandasai/agent/base.py

```diff
@@ -260,7 +305,57 @@ def chat(self, query: str, output_type: Optional[str] = None):
                 query, output_type, self.conversation_id, self.last_prompt_id
             )
 
-            return self.chat_pipeline.run(pipeline_input)
+            return self.generate_code_pipeline.pipe(
```

Comment:
@gventuri Also i think we should keep agent as it and concatenate two pipelines inside GenerateChatPipeline using | operator because for agent the whole pipeline needs to executed. What do you think?